openmetadata-ui/src/main/resources/ui/src/styles/components/button.less

```diff
@@ -116,25 +150,26 @@ button {
   background: transparent;
 }
 
+.ant-tooltip-disabled-compatible-wrapper.bordered {
+  .ant-btn-default {
+    .icon-button-default();
+  }
+
+  .ant-btn-sm {
+    .icon-button-sm();
+  }
+}
+
 .ant-btn.ant-btn-icon-only.bordered {
   border: 1px solid @grey-300;
   background-color: @white;
 
-  &.ant-btn-sm {
-    width: 20px;
-    height: 20px;
-    border-radius: @border-radius-base;
-    padding: 0;
-    display: inline-flex;
-    align-items: center;
-    justify-content: center;
-    line-height: 1;
+  &.ant-btn-default {
+    .icon-button-default();
+  }
```

Comment:
Why we are changing default button style, it will impact all the places

Reply:
This will only apply for element that has .border class and  only icon